retrieval_service/app/routes.py

```diff
@@ -69,15 +69,15 @@ async def get_airport(
 ):
     ds: datastore.Client = request.app.state.datastore
     if id:
-        results = await ds.get_airport_by_id(id)
+        results, sql = await ds.get_airport_by_id(id)
     elif iata:
-        results = await ds.get_airport_by_iata(iata)
+        results, sql = await ds.get_airport_by_iata(iata)
     else:
         raise HTTPException(
             status_code=422,
             detail="Request requires query params: airport id or iata",
         )
-    return results
+    return {"results": results, "sql": sql, "error": None}
```

Comment:
Why specify error? It looks like it's always None?

Reply:
I added this incase we eventually want to display error and improve the frontend error display. I'll remove this for now and add it when we implement that.